chat-plugins/scavengers.js

```diff
@@ -1,303 +1,603 @@
-/**
-* Scavenger hunts plugin. Only works in a room with the id 'scavengers'.
-* This game shows a hint. Once a player submits the correct answer, they move on to the next hint.
-* You finish upon correctly answering the third hint.
-* In an official hunt, the first three to finish within 60 seconds achieve blitz.
-*/
```

Comment:
The line about the blitz isn't valid.

If something __were__ to be put there, wouldn't the MIT stuff usually make more sense, since that's whats on most of the other files.